store/src/java-test/com/zimbra/cs/filter/NotifyTest.java

```diff
@@ -112,4 +112,37 @@ public void filterValidToField() {
 
     }
 
+    @Test
+    public void testNotifyMailtoWithHeaderVariable() {
+        String sampleMsg = "from: abc@zimbra.com\n"
+                + "Subject: Hello\n"
+                + "to: test@zimbra.com\n";
+        String filterScript = "require [\"enotify\", \"variables\"];\n"
+            + "set \"to\" \"nick\";\n"
+            + "if anyof (header :contains [\"Subject\"] \"Hello\") {\n"
+            + "notify \"test2@zimbra.com\" \"\" \"${SUBJECT} ${to}\"\n"
```

Comment:
The following use case breaks the backward compatibility; when the user migrates the filter rule which contains the legacy notify action with ${from}, then he/she adds the 'set' action on it to set some value to the variable name of "from".
But as talking with Rupali, and as considering the possibility of our current customers migrating the legacy notify action (which is low) to the 8.7.x, I was convinced that it can be handled by the well-warned documentation.